js/private/js_run_binary.bzl

```diff
@@ -127,6 +128,17 @@ def js_run_binary(
 
             See https://docs.bazel.build/versions/main/be/common-definitions.html#common.tags for useful keys.
 
+        patch_node_fs_api: Patch the to Node.js `fs` API (https://nodejs.org/api/fs.html) for this node program
```

Comment:
naming nit: "api" doesn't seem necessary here, the user isn't coding against it. WDYT about just `patch_node_fs`?

Reply:
sgtm